src/main/java/mate/academy/service/UserService.java

```diff
@@ -8,13 +8,16 @@ public class UserService {
             new User("alice@i.ua", "1234")
     };
 
-    /**
-     * Find user by email. All users are stored in <code>private static final User[] users</code>
-     * @param email - the input parameter
-     * @return - user if his email is equal to passed email.
-     * Return <code>null</code> if there is no suitable user
-     */
+    public static User[] getUsers() {
+        return users;
+    }
+
     public User findByEmail(String email) {
+        for (int i = 0; i < UserService.getUsers().length; i++) {
```

Comment:
you can user for each and find compare email `user.getEmail()`
```suggestion
        for (User user : users) {
```